crates/bevy_ui/src/ui_node.rs

```diff
@@ -184,12 +247,17 @@ impl Default for Display {
     }
 }
 
+/// Defines how flexbox items are ordered within a flexbox
 #[derive(Copy, Clone, PartialEq, Debug, Serialize, Deserialize, Reflect)]
 #[reflect_value(PartialEq, Serialize, Deserialize)]
 pub enum FlexDirection {
+    /// Same way as text direction along the main axis
     Row,
+    /// Same way as text direction along the cross axis
```

Comment:
This describes what we should be doing, but as you documented somewhere else we always use bottom to top for Column and top to bottom for ColumnReverse, ignoring the text direction.